test/e2e/sample.spec.js

```diff
@@ -8,12 +8,18 @@ describe("Protractor helper", () => {
   const inputFile = element(by.className("inputfile"));
   const shortenButton = element(by.id("shorten"));
   const expandButton = element(by.id("expand"));
+  const alertButton = element(by.id("alertButton"));
 
   const nonExistingElement = element(by.id("foo"));
 
   beforeEach(() => browser.get(""));
 
   describe("waits", () => {
+    it("waitForAlertToBePresent", () => {
+      helper.click(alertButton);
+      helper.waitForAlertToBePresent(5000);
```

Comment:
```suggestion
      helper.waitForAlertToBePresent();
```

This first test should test that it works without passing the optional timeout argument, just to follow the order we've been using for other similar tests.

Reply:
We also need to test if it works when passing a timeout different from the default one. See other tests for examples.

Reply:
You will also need to accept the dialog that the test will trigger.

I suggest using something like:

```js
const alertDialog = browser.switchTo().alert();
alertDialog.accept();
```

Reply:
Done